### PR TITLE
fix wrong hyperlink to distribute_strategy.ipynb

### PR DESCRIPTION
**Pull request by Atharv24**
I noticed the hyperlink was throwing a 404 on the website. This should work.

### CODE REVIEW

**Comment by Atharv24** [comment]
Signed it! 💃

File: site/en/r2/guide/using_gpu.ipynb [hunk 1]
```diff
@@ -70,7 +70,7 @@
         "\n",
         "Note: Use the `tf.test.is_gpu_available` function to confirm that TensorFlow is using the GPU.\n",
         "\n",
-        "The simplest way to run on multiple GPUs, on one or many machines, is using [Distribution Strategies](../distribute_strategy.ipynb)\n",
+        "The simplest way to run on multiple GPUs, on one or many machines, is using [Distribution Strategies](../guide/distribute_strategy.ipynb)\n",
```
Comment (yashk2810) [inline site/en/r2/guide/using_gpu.ipynb]: SInce its in the same directory, it works with [distribute_strategy.ipynb] too :)

Comment (Atharv24) [reply site/en/r2/guide/using_gpu.ipynb]: Its throwing 404 on the main website and on GitHub too

Comment (Atharv24) [reply site/en/r2/guide/using_gpu.ipynb]: :o thanks for the tip!

Comment (yashk2810) [reply site/en/r2/guide/using_gpu.ipynb]: Right. Changing it to [Distribution Strategies](distribute_strategy.ipynb) will work too.

**Pull request merged**